SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/UNR-4028/SpatialAuthorityTest.cpp

```diff
@@ -0,0 +1,358 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "SpatialAuthorityTest.h"
+#include "Engine/World.h"
+#include "Net/UnrealNetwork.h"
+#include "SpatialAuthorityTestActor.h"
+#include "SpatialAuthorityTestGameMode.h"
+#include "SpatialAuthorityTestGameState.h"
+#include "SpatialAuthorityTestReplicatedActor.h"
+#include "SpatialFunctionalTestFlowController.h"
+#include "SpatialAuthorityTestFlowController.h"
+
+/** This Test is meant to check that HasAuthority() rules are respected on different occasions. We check
+  * in BeginPlay and Tick, and in the following use cases:
+  *		- replicated level actor
+  *		- non-replicated level actor
+  *		- dynamic replicated actor (one time with spatial authority and another without)
+  *		- dynamic non-replicated actor
+  *		- GameMode (which only exists on Servers)
+  *		- GameState
+  * Keep in mind that we're assuming a 1x2 Grid Load-Balancing Strategy, otherwise the ownership of
+  * these actors may be something completely different (specially important for actors placed in the Level).
+  * You have some flexibility to change the Server1/2Position properties to test in different Load-Balancing Strategies.
```

Comment:
Super NIT
```suggestion
  * You have some flexibility to change the Server1/2 Position properties to test in different Load-Balancing Strategies.
```

Reply:
That's together because it's the name of the variable

Reply:
Oh, that makes sense, thank you!